sinks/xray/xray.go

```diff
@@ -71,35 +81,78 @@ type XRaySpanSink struct {
 	sampleThreshold uint32
 	commonTags      map[string]string
 	annotationTags  map[string]struct{}
-	log             *logrus.Logger
+	log             *logrus.Entry
 	spansDropped    int64
 	spansHandled    int64
+	name            string
 	nameRegex       *regexp.Regexp
 }
 
 var _ sinks.SpanSink = &XRaySpanSink{}
 
-// NewXRaySpanSink creates a new instance of a XRaySpanSink.
-func NewXRaySpanSink(daemonAddr string, sampleRatePercentage float64, commonTags map[string]string, annotationTags []string, log *logrus.Logger) (*XRaySpanSink, error) {
+// TODO(yeogai): Remove this once the old configuration format has been
+// removed.
+func MigrateConfig(conf *veneur.Config) error {
```

Comment:
nit: if you're always returning nil, you can drop the return type.

Reply:
Thanks, updated